CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 * [STREAM-207](https://inindca.atlassian.net/browse/STREAM-207) - [STREAM-207] handle ice candidates received before the offer (sdpOverXmpp only)
+* [STREAM-326](https://inindca.atlassian.net/browse/STREAM-326) - Only delete pending sessions when handling Jingle session-initiate to avoid emitting two sessions (partial reversion of d1251b22c1b82ddd0b514a5384a89510082dda26).
```

Comment:
@zservies @maxwellmooney13 Thoughts on this? I was a little inclined to change the note on PCM-2081 above since this hasn't released yet (it seems less confusing in the log to avoid it altogether), but on the other hand, that would lose the ticket reference. This seemed like a reasonable compromise, but I'm open to ideas.